README.md

```diff
@@ -1,241 +1,158 @@
-     Running `target/release/mint-client-cli cfg info`
-Jun 15 14:57:22.066  INFO mint_client_cli: We own 14 coins with a total value of 9500000 msat
-Jun 15 14:57:22.066  INFO mint_client_cli: We own 5 coins of denomination 100000 msat
-Jun 15 14:57:22.066  INFO mint_client_cli: We own 9 coins of denomination 1000000 msat
-```
-
-The `spend` subcommand allows sending tokens to another client. This will select the smallest possible set of the client's coins that represents a given amount. The coins are base64 encoded and printed to stdout.
-
-```
-minimint $ cargo run --bin mint-client-cli --release -- cfg spend 400000
-    Finished release [optimized] target(s) in 0.12s
-     Running `target/release/mint-client-cli cfg spend 400000`
-AQAAAAAAAACghgEAAAAAAAQAAAAAAAAAA7mGus9L4ojsIsctFK7oNz5s4ozZe3pVa0S1jZ3XvSnMMAAAAAAAAACFjOG3a4vlxBOCa9fYD6qWIM2JhH9vitG0
-DXQhd9KhGYKheKADLOVXgZwOQDX0NheGP5fFEMYOfidY1FPXB1qRhrEiZKh3YVb2i922uUHoggOsZhrLpk4EGCJjuT1QUWpO8HZ9WOxD4oUv6nPNVQnKvDAA
-AAAAAAAAoXKhtm/0w8pFz7CN6xcEQUnukrNcfhc/NtRita1vvZDyX/NBiSmHZVyWx8WEloclIw0A8ljJhp+b517c1LsLJ5Z6Issf9QcV/hwAgY/RJo4DRGWD
-IDyyyBYXxRbFuTZoDaTR3TM/49m41Bl7/CPVz98wAAAAAAAAAJOjsZSwWrBUXt+OsojEkxRbqn8KAJrz1TTQkNrdlEiaSRqjx+YCfET3HwL3j26s2clhRugM
-rRj6oMC6wKoZz0jCuS5i8faLRHGZp3AMR1/xAvMglQZ9zMEDdDd7dcxwp9WpR6JfdAUJku3EGQ/FUXaQMAAAAAAAAACUXn9s935ruZ5jA5o5aNf1u/smH4TN
-+qO8jMHVf6Zzh22P5jJvhWdX62s7kftXTa9AKeiC0I4QxWdWVK4JTLnE62GzGLQqQyEkne3Pn/Pm1g==
-```
-
-A receiving client can now reissue these coins to claim them and avoid double spends:
+$ mint_client reissue AQAAAAAAAABAQg8AAA...
+$ mint_client fetch
 
-```
-minimint $ cargo run --bin mint-client-cli --release -- cfg reissue AQAAAAAAAACghgE…
-    Finished release [optimized] target(s) in 0.13s
-     Running `target/release/mint-client-cli cfg reissue AQAAAAAAAACghgE…
-Jun 15 15:01:47.027  INFO mint_client_cli: Starting reissuance transaction for 400000 msat
-Jun 15 15:01:47.040  INFO mint_client_cli: Started reissuance 47d8f08710423c1e300854ecb6463ca6185e4b3890bbbb90fd1ff70c72e1ed18, please fetch the result later
-minimint $ cargo run --bin mint-client-cli --release -- cfg fetch
-    Finished release [optimized] target(s) in 0.12s
-     Running `target/release/mint-client-cli cfg fetch`
-Jun 15 15:02:06.264  INFO mint_client_cli: Fetched coins from issuance 47d8f08710423c1e300854ecb6463ca6185e4b3890bbbb90fd1ff70c72e1ed18
+INFO mint_client_cli: Fetched coins issuance=5b1ac4e9604...
 ```
 
 ### Using the gateway
 
-To pay a Lightning invoice using ecash tokens via the gateway we must run the gateway itself:
-
-```shell
-cargo run --bin ln_gateway --release cfg
-```
-
 Generate a Lightning invoice from LN2, our non-gateway lightning node:
 
 ```shell
-# Make sure your peg-in created coins of small enough tiers. For now the client is not able to spend without exact change.
-lightning-cli --network regtest --lightning-dir=ln2 invoice 100000 test test 10m
+$ INVOICE="$(ln2 invoice 100000 test test 1m | jq -r '.bolt11')"
```

Comment:
When we mention the nix and rust dependencies, should we also mention that "this tutorial will also use `jq`, but it's not a requirement of minimint" or something like that?

Reply:
I think that using `jq` is good because you can just copy-paste everything

Reply:
I see you update and yes that looks better 👍